src/pages/Home/home.css

```diff
@@ -19,9 +19,11 @@
 .ni-wa {
   font-size: 6rem;
   line-height: 1rem;
+  color: #172f4a;
 }
```

Comment:
Its Ok to do in a different PR 
we can define css variables for the values of styles that we're going to reuse throughout the app, so editing would be easier